drivers/serial/uart_pl011.c

```diff
@@ -519,26 +529,26 @@ static void pl011_irq_config_func_1(struct device *dev)
 	shared_irq_isr_register(shared_irq_dev, (isr_t)pl011_isr, dev);
 	shared_irq_enable(shared_irq_dev, dev);
 #else
-	IRQ_CONNECT(DT_PL011_PORT1_IRQ_TX,
-		    DT_PL011_PORT1_IRQ_PRI,
+	IRQ_CONNECT(DT_INST_1_ARM_PL011_IRQ_TX,
+		    DT_INST_1_ARM_PL011_IRQ_TX_PRIORITY,
 		    pl011_isr,
-		    DEVICE_GET(pl011_port_1),
+		    DEVICE_GET(pl011_port_0),
 		    0);
-	irq_enable(DT_PL011_PORT1_IRQ_TX);
+	irq_enable(DT_INST_1_ARM_PL011_IRQ_TX);
 
-	IRQ_CONNECT(DT_PL011_PORT1_IRQ_RX,
-		    DT_PL011_PORT1_IRQ_PRI,
+	IRQ_CONNECT(DT_INST_1_ARM_PL011_IRQ_RX,
+		    DT_INST_1_ARM_PL011_IRQ_RX_PRIORITY,
 		    pl011_isr,
-		    DEVICE_GET(pl011_port_1),
+		    DEVICE_GET(pl011_port_0),
```

Comment:
Don't think this should be changed

Reply:
thanks, good catch.

Reply:
The PR got merged but the issue seems to be still unresolved. Do we need a follow up patch?

Reply:
Yes, sorry about that.  Pushed PR #23488 for that.